### PR TITLE
Bugfix - talks list view  

### PR DESCRIPTION
**Pull request by HedicGuibert**
This fixes a bug where, if a user was giving the same talk multiple times alone, it would add a / after his name even if he's alone in his column.
This was caused by the `{% if not loop.last %}/{% endif %}` and the unique filtering we apply. Twig sees there are 2 submits, so loops twice and adds a / at the end of the first, which is not end of loop. But it prevents the second from being displayed because the user is not unique so it just leaves the /.

So we send submits with the same author alone only once.

EDIT : Ok so I ended up doing quite a bit more than fixing a bug. The bug is fixed by making the code way simpler, and it also rewrites the fixtures and add tests. It should be way better now.

### CODE REVIEW

File: tests/TalkTest.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Tests\Entity;
+
+use App\Entity\Submit;
+use App\Entity\Talk;
+use PHPUnit\Framework\TestCase;
+
+class TalkTest extends TestCase
```
Comment (lyrixx) [inline tests/TalkTest.php:9]: This file is not in the right directory. It should follow the same convention as the `src/` dir

**Review by lyrixx — COMMENTED** [review]
This is a very good start 👍🏼

**Comment by HedicGuibert** [comment]
I updated and squashed to make it less messy.

File: src/Entity/Talk.php [hunk 2]
```diff
@@ -101,6 +101,24 @@ public function removeSubmit(Submit $submit): self
         return $this;
     }
 
+    public function getUniqueUsers()
```
Comment (lyrixx) [inline src/Entity/Talk.php:104]: missing type hint

File: templates/conferences/list.html.twig [hunk 3]
```diff
@@ -69,7 +69,7 @@
                 </div>
 
                 {% set activeTab = activeTab|default(liveConferences|length > 0 ? 'live' : futureConferences|length > 0 ? 'future' : 'past') %}
-
+
```
Comment (lyrixx) [inline templates/conferences/list.html.twig:72]: oups

File: tests/Entity/TalkTest.php [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            }
+
+            $talk->addSubmit($newSubmit);
+        }
+
+        $result = $talk->getUniqueUsers();
+        self::assertSame($expectedAmount, \count($result));
+    }
+
+    public function provideUsers()
```
Comment (lyrixx) [inline tests/Entity/TalkTest.php:45]: missing type hint (`Generator`)

File: tests/Entity/TalkTest.php [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+                $newUser->setName($user);
+
+                $newSubmit->addUser($newUser);
+            }
+
+            $talk->addSubmit($newSubmit);
+        }
+
+        $result = $talk->getUniqueUsers();
+        self::assertSame($expectedAmount, \count($result));
```
Comment (lyrixx) [inline tests/Entity/TalkTest.php:42]: There is a method in phpunit for such assertion (`assetCount`)

**Review by lyrixx — COMMENTED** [review]
Except some minor comments, this is nice 👍🏼 Well done

**Review by lyrixx — APPROVED** [review]

**Pull request merged**